### DISCUSSION TITLE
app.storage.user does not work from within iframe

### DISCUSSION THREAD

**Discussion by jeffective** [zauberzeug/nicegui#1645] [Q&A] [answered]
### Question

I am using app.storage.user similarly to the authentication example however it doesnt seem to be working from within an iframe (my webpage is being viewed from an iframe)

**Comment by falkoschindler** [comment, marked as answer]
Can you, please, show a [minimal reproducible code example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) and give more information about how it is not working? That would allow us to help more efficiently. Thanks!

**Reply by jeffective** [reply]
i wasnt able to reproduce the issue, so thats good I guess :)
